src/rpc/misc.cpp

```diff
@@ -611,6 +656,7 @@ static const CRPCCommand commands[] =
     { "util",               "getdescriptorinfo",      &getdescriptorinfo,      {"descriptor"} },
     { "util",               "verifymessage",          &verifymessage,          {"address","signature","message"} },
     { "util",               "signmessagewithprivkey", &signmessagewithprivkey, {"privkey","message"} },
+    { "util",               "listindices",            &listindices,            {} },
```

Comment:
unrelated style-nit: Maybe use the "new" constructor that takes an RpcMethodFnType . See https://doxygen.bitcoincore.org/class_c_r_p_c_command.html#afa39f5bda9319f3b91c6b38bbc7c7434 and #19528